flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -329,6 +334,10 @@ static constexpr IntrinsicHandler handlers[]{
     {"modulo", &I::genModulo},
     {"nint", &I::genNint},
     {"present", &I::genPresent, {{{"a", asInquired}}}, /*isElemental=*/false},
+    {"rrspacing", 
+     &I::genRRSpacing,
+     {{{"x", asValue}}},
```

Comment:
Nit: You do not need the:
```
  {{{"x", asValue}}},
     /*isElemental=*/true},
```
part since this is the default, but it is not wrong to have it.

Reply:
@jeanPerier Is the /*isElemental=*/true also the default?